src/vos/vos_pool.c

```diff
@@ -88,6 +88,131 @@ vos_meta_readv(struct umem_store *store, struct umem_store_iod *iod, d_sg_list_t
 	return vos_meta_rwv(store, iod, sgl, false);
 }
 
+
+#define META_READ_BATCH_SIZE (1024 * 1024)
+
+struct meta_load_control {
+	ABT_mutex		mlc_lock;
+	ABT_cond		mlc_cond;
+	int			mlc_rc;
+	int			mlc_inflights;
+	int			mlc_wait_finished;
+};
+
+struct meta_load_arg {
+	daos_off_t		  mla_off;
+	uint64_t		  mla_read_size;
+	char		         *mla_start;
+	struct umem_store        *mla_store;
+	struct meta_load_control *mla_control;
+};
+
+#define META_READ_QD_NR	4
```

Comment:
Can we get better number with larger QD?

Reply:
No..